pkgs/applications/networking/mailreaders/afew/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, pythonPackages, notmuch }:
+
+let
+  pname = "afew";
+  version = "1.2.0";
+in
+
+pythonPackages.buildPythonApplication {
+  inherit pname version;
+
+  src = pythonPackages.fetchPypi {
+    inherit pname version;
+    sha256 = "121w7bd53xyibllxxbfykjj76n81kn1vgjqd22izyh67y8qyyk5r";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [
+    setuptools_scm pythonPackages.notmuch chardet
+  ] ++ stdenv.lib.optional (!pythonPackages.isPy3k) subprocess32;
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = "${version}";
```

Comment:
preBuild is this likely a better place for this one

Reply:
seems like this isn't required anymore at all, as we're fetching from pypi - and setuptools_scm is able to figure out the version on its own